site/docs/guide/files.md

```diff
@@ -5,107 +5,112 @@ next: ./games.md
+This method enables you to download the file by constructing a special, temporary, URL.
+Note that this URL is only guaranteed to be valid for 60 minutes, after which it may expire. In this case, you can simply call `getFile` again.
 
-When a bot **sends** a message, it can specify a `file_id` that is has seen before.
-This will then send the identified file.
-(You can also upload your own files, [scroll down](#sending-files) to see how.)
-You can reuse the same `file_id` as often as you want, so you could send the same file to five different chats, always with the same `file_id`.
-Make sure to use the correct method—for instance, you cannot use a `file_id` that identifies a photo when calling [`sendVideo`](https://core.telegram.org/bots/api#sendvideo).
+When a bot **sends** a message, it can specify a `file_id` that it has seen before.
+This will allow it to send the identified file, without needing to upload the data for it.
+(To see how to upload your own files, [scroll down](#sending-files).)
+You can reuse the same `file_id` as often as you want, so you could send the same file to five different chats, using the same `file_id`.
+However, you must make sure to use the correct method—for example, you cannot use a `file_id` that identifies a photo when calling [`sendVideo`](https://core.telegram.org/bots/api#sendvideo).
 
 Every bot has its own set of `file_id`s for the files that it can access.
-You cannot give the `file_id` to your friend's bot to download it there, as another bot will use a different identifier for the same file.
-This implies that you cannot simply guess a `file_id` and access some random person's file, because Telegram keeps track of what `file_id`s are valid for your bot.
+You cannot reliably use a `file_id` from your friend's bot, to access a file with _your_ bot. Each bot will use different identifiers for the same file.
+This implies that you cannot simply guess a `file_id` and access some random person's file, because Telegram keeps track of which `file_id`s are valid for your bot.
 
-On the other hand, it can happen that the same file is identified by different `file_id`s even for the same bot.
-That means that you cannot reliably compare `file_id`s to check if two files are the same.
-In cases where you do need to identify the same file over time and across bots, you should use the value for `file_unique_id` that your bot receives along with every `file_id`.
-The `file_unique_id` cannot be used to download files.
+::: warning Using Foreign file_ids
+Note that in some cases it _is_ technically possible that a `file_id` from another bot seems to work correctly.
+**However**, using a foreign `file_id` like this is dangerous as it can stop working at any time, without warning.
+So, always ensure that any `file_id`s you use were originally for your bot.
+:::
+
+On the other hand, it _is_ possible that a bot eventually sees the same file identified by different `file_id`s.
```

Comment:
This emphasis doesn't make sense to me

Reply:
Style, but here it is not serious enough to dispute. :)
```suggestion
On the other hand, it is possible that a bot eventually sees the same file identified by different `file_id`s.
```